homeassistant/components/yolink/binary_sensor.py

```diff
@@ -20,6 +20,7 @@
     ATTR_COORDINATORS,
     ATTR_DEVICE_DOOR_SENSOR,
     ATTR_DEVICE_LEAK_SENSOR,
+    ATTR_DEVICE_LEAK_VIBRATION_SENSOR,
```

Comment:
Is it a vibration sensor or a leak sensor or both?

The name seems to suggest it is both.

Reply:
sorry, i notice that, i'm pushing the new commits.